src/lib/coingecko/utils.ts

```diff
@@ -118,3 +134,19 @@ export const getCoingeckoMarkets = async (
 
   return all.filter(mover => Boolean(mover.assetId))
 }
+
+export const getCoingeckoSupportedChainIds = () => {
```

Comment:
As a function to avoid module exports shenanigans with programmatic flag nova spread